README.md

```diff
@@ -1,3 +1,17 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
Please revert changes to this file.  Our docs magician (@abrennan89) is hard at work reworking it and she would rather not have conflicting changes to deal with.

Reply:
Done